app/controllers/passengers_controller.rb

```diff
@@ -48,8 +48,11 @@ def destroy
   private
 
   def passenger_params
-    params.require(:passenger).permit(:name, :address, :email, :phone,
-      :wheelchair, :active, :permanent, :expiration, :note)
+    permitted_params = params.require(:passenger).permit(:name, :address, :email, :phone,
+        :wheelchair, :active, :permanent, :expiration, :note)
+    unless @current_user.admin?
+      permitted_params = permitted_params.except(:active, :permanent)
+    end
```

Comment:
What I'm getting at is that it doesn't return `permitted_params`, but seems like it should.